docs/reference/run.md

```diff
@@ -984,6 +984,7 @@ or override the Dockerfile's exposed defaults:
                    format: ip:hostPort:containerPort | ip::containerPort | hostPort:containerPort | containerPort
                    Both hostPort and containerPort can be specified as a range of ports.
                    When specifying ranges for both, the number of container ports in the range must match the number of host ports in the range. (e.g., `-p 1234-1236:1234-1236/tcp`)
+                   When specifying a range for hostPort only, the containerPort must not be a range.  In this case the container port is published somewhere within the specified hostPort range. (e.g., `-p 1234-1236:1234/tcp`)
```

Comment:
@dkjer Thank you for your contribution.  This is great information and I think expanding it will help readers get this information more easily.  Also, I'd like it if we could put this information in a table.  The table is beyond your scope, if you are willing to add it, good. If not I can carry the work for you.

--> https://gist.github.com/moxiegirl/8fa6820db70a011a8414

<table>
  <tr>
    <th>Option</th>
    <th>Description</th>
  </tr>
  <tr>
    <td><code>--expose=[]</code></td>
    <td>Expose a port or a range of ports from the container,without publishing it to your host</td>
  </tr>
  <tr>
    <td><code>-P=false</code></td>
    <td>Publish all exposed ports to the host interfaces.</td>
  </tr>
  <tr>
    <td><code>-p=[]<code></td>
    <td>
      <p>
      Publish a container᾿s port or a range of ports to the host. The value's format can be any of:    
      <ul>
      <li> <code>ip:hostPort:containerPort</code> 
      <li> <code>ip::containerPort</code> 
      <li> <code>hostPort:containerPort</code>
      <li> <code>containerPort</code>
      </ul>
      </p>
      <p>
      Use the <code>ip:hostPort:containerPort</code> or <code>ip::containerPort</code> formats
      if you want to be sure to bind a specific interface on the host side.
      </p>
      <p>
      You can specify <i>both</i> the <code>hostPort</code> and <code>containerPort</code> as port ranges.
      When specifying ranges for both, the number of values in the ranges
      must match, for example:<code>-p 2000-3000:2500-3500/tcp</code> The total number of
      values are both 1000.
      </p>
      <p>
      You can also specify a range for <code>hostPort</code> alone and a single
      value for the container:<code>-p 1234-1236:2000/tcp</code> In this case,
      the <code>containerPort</code> is mapped to a value somewhere within the
      given <code>hostPort</code> range. Use the <code>docker port</code>
      command to see the actual mapping.
      </p>
      </td>
  </tr>
  <tr>
    <td><code>--link=""</code></td>
    <td>Add link to another container (<code>id:alias</code>, <code>name:alias</code>, <code>name</code>, <code>id</code>)</td>
  </tr>
</table>